src/Core/System/Snippet/SnippetFixer.php

```diff
@@ -15,7 +21,31 @@ public function __construct(private readonly SnippetFileHandler $snippetFileHand
     {
     }
 
-    public function fix(MissingSnippetCollection $missingSnippetCollection): void
+    /**
+     * @deprecated tag:v6.8.0 reason:new-optional-parameter - Will get a second parameter `$invalidPluralization`
+     */
+    public function fix(MissingSnippetCollection $missingSnippetCollection /* , InvalidPluralizationCollection $invalidPluralization */): void
+    {
+        /** @var InvalidPluralizationCollection $invalidPluralization */
+        $invalidPluralization = \func_num_args() === 2 ? func_get_arg(1) : [];
```

Comment:
if the parameter is missing, this should probably be an empty collection 

```suggestion
        $invalidPluralization = \func_num_args() === 2 ? func_get_arg(1) : new InvalidPluralizationCollection();
```

Reply:
good catch!